libcloudforensics/providers/gcp/internal/metrics.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Google Cloud Metrics functionality."""
+
+import datetime
+from typing import TYPE_CHECKING, Dict
+
+from libcloudforensics.providers.gcp.internal import common
+
+if TYPE_CHECKING:
+  import googleapiclient
+
+
+class GoogleCloudMetrics:
+  """Class to call Google Metric APIs.
+
+  https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.timeSeries
+
+  Attributes:
+    project_id: Project name.
+    gcm_api_client: Client to interact with Metrics APIs.
+  """
+  CLOUD_METRICS_API_VERSION = 'v3'
+
+  def __init__(self, project_id: str) -> None:
+    """Initialize the GoogleCloudMetrics object.
+
+    Args:
+      project_id (str): The name of the project.
+    """
+
+    self.gcm_api_client = None
+    self.project_id = project_id
+
+  def GcmApi(self) -> 'googleapiclient.discovery.Resource':
+    """Get a Google Cloud Metrics service object.
+
+    Returns:
+      googleapiclient.discovery.Resource: A Google Cloud Metrics service object.
+    """
+    if self.gcm_api_client:
+      return self.gcm_api_client
+    self.gcm_api_client = common.CreateService(
+        'monitoring', self.CLOUD_METRICS_API_VERSION)
+    return self.gcm_api_client
+
+  def ActiveServices(self, timeframe: int = 30) -> Dict[str, int]:
+    """List active services in the project (default: last 30 days).
+
+    Args:
+      timeframe (int): The number (in days) for which to measure activity.
```

Comment:
add "Optional."
EX: timeframe (int): Optional. The number (in days) for which to measure activity. Default is 30 days.